### DISCUSSION TITLE
Error while delete Cache Folder

### DISCUSSION THREAD

**Discussion by cracksilver** [automadcms/discussions#47] [Q&A]
hi there
I got the following error when I try to delete the cache folder
![grafik](https://github.com/user-attachments/assets/61cdf1d5-cc0c-4c43-96b8-8cd5d0a8f97e)
What does that mean?

**Comment by marcantondahmen** [comment]
This should be fixed with the latest alpha release.

**Comment by cracksilver** [comment]
thx. it works now.
